### PR TITLE
tracing-futures: prepare 0.2.1

### PR DESCRIPTION
**Pull request by davidbarsky**

### CODE REVIEW

File: tracing-futures/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# 0.2.0 (Nov XX, 2019)
+# 0.2.1
+
+- docs: add `doc-cfg` attributes to tracing, tracing-core, tracing-futures, tracing-subscriber (#523, #525)
```
Comment (hawkw) [inline tracing-futures/CHANGELOG.md]: I don't think it's necessary to mention changes to other crates in the `tracing-futures` changelog. 

Also, for the other crates where we addded the doc-cfg attributes, I phrased this like 
```markdown
### Added

- ...
- API documentation now shows which features are required by feature-flagged items (#523)

Comment (davidbarsky) [reply tracing-futures/CHANGELOG.md]: will add.

File: tracing-futures/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
-# 0.2.0 (Nov XX, 2019)
+# 0.2.1
```
[the remark this answers is not in the document]
Comment (davidbarsky) [reply tracing-futures/CHANGELOG.md]: d'oh, i thought i added that. fixing.

**Comment by hawkw** [comment]
@davidbarsky any updates on this?

**Comment by davidbarsky** [comment]
@hawkw pushed with my first git tag usage (ever!). Let me know if you want to change anything else.

**Comment by hawkw** [comment]
@davidbarsky done! https://crates.io/crates/tracing-futures/0.2.1

**Pull request merged**